assets/js/components/frontpage/overrides/RecordsList.js

```diff
@@ -18,60 +18,10 @@ import {
   Button,
   Message,
 } from "semantic-ui-react";
-import { RecordsResultsListItem } from "./RecordsResultsListItem";
 
-export class RecordsList extends Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      data: { hits: [] },
-      isLoading: false,
-      error: null,
-    };
-  }
-
-  componentDidMount() {
-    this.fetchData();
-  }
-
-  componentWillUnmount() {
-    this.cancellableFetch && this.cancellableFetch.cancel();
-  }
-
-  fetchData = async () => {
-    const { fetchUrl } = this.props;
-    this.setState({ isLoading: true });
-
-    this.cancellableFetch = withCancel(
-      http.get(fetchUrl, {
-        headers: {
-          Accept: "application/vnd.inveniordm.v1+json",
-        },
-      })
-    );
-
-    try {
-      const response = await this.cancellableFetch.promise;
-      this.setState({ data: response.data.hits, isLoading: false });
-    } catch (error) {
-      console.error(error);
-      this.setState({
-        error: "Unable to load records",
-        isLoading: false,
-      });
-    }
-  };
-
-  render() {
-    const { isLoading, data, error } = this.state;
-    const { title } = this.props;
-
-    const listItems = data.hits?.map((record) => {
-      return <RecordsResultsListItem result={record} key={record.id} />;
-    });
-
-    return (
+export const CDSRecordsList = ({ isLoading, error, title, listItems }) => {
+  return (
+    !isEmpty(listItems) && (
```

Comment:
wouldn't it better if we implement this globally so that other instances (f.e. Zenodo) can also benefit from this fix ?

Reply:
This was implemented on invenio-app-rdm, so its global. I left the if condition inside the overridable jsx in case in some instance we want to display a message instead of completely removing the section.

Reply:
could you show me there exactly this if statement is implemented in app_rdm ?

Reply:
https://github.com/inveniosoftware/invenio-app-rdm/pull/2007/files#diff-bc5bd180fc17fe701ef603df6390ca4466032be536713855b1fb469b13c2e510R72